spec/menu_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'menu'
+
+
+ describe "menu" do
+      it "is initialised with a hash of dishes by default" do
+          menu1 = Menu.new
+        expect(menu1.food_list).to eq({:Rice=>2.5, :Salad=>3.4, :Cheese=>2.9, :Samon=>1.5, :Bacon=>2.45 })
```

Comment:
Metrics/LineLength: Line is too long. [106/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)